src/rust/engine/process_execution/src/remote.rs

```diff
@@ -133,7 +133,14 @@ impl CommandRunner {
     let store_use_tls = store_address.starts_with("https://");
 
     let tls_client_config = if execution_use_tls || store_use_tls {
-      Some(grpc_util::create_tls_config(root_ca_certs)?)
+      Some(
+        grpc_util::TlsConfig {
```

Comment:
Maybe provide a constructor on `TlsConfig` to take the root certs and initialize everything else (or mabe rely on `TlsConfig::default` to init `mtls` and `certificate_check`)? Would reduce boilerplate in code not in `grpc_util`.